src/target/xtensa/xtensa.c

```diff
@@ -938,10 +999,11 @@ int xtensa_deassert_reset(struct target *target)
 	struct xtensa *xtensa = target_to_xtensa(target);
 
 	LOG_TARGET_DEBUG(target, "halt=%d", target->reset_halt);
-	if (target->reset_halt)
+	if (target->reset_halt) {
```

Comment:
No need to add parentheses for the single line statements. Probably came from auto format tool. I'll fix it.